integrations/purchase/nordic-api-gateway/nag-categories.js

```diff
@@ -45,7 +44,7 @@ const categoryToCategory = {
   'Auto Loan etc.': null,
   Baby: PURCHASE_CATEGORY_OTHER_PERSONAL_EFFECTS,
   'Bank Fees': PURCHASE_CATEGORY_FINANCIAL_SERVICES,
-  'Basic Expenses': PURCHASE_CATEGORY_FOOD,
+  'Basic Expenses': null,
```

Comment:
Why remove this one? It should point to `Food`, which is in footprints.yml?

![image](https://user-images.githubusercontent.com/3296643/102766575-9df5bd00-437e-11eb-9b0c-5ed9a3af7dfa.png)


![image](https://user-images.githubusercontent.com/3296643/102766555-97ffdc00-437e-11eb-9711-9cdb1860fea9.png)

Reply:
The test didn’t find a carbon intensity. I’ll check. :)

Reply:
So the issue seems to be that https://github.com/tmrowco/bloom-contrib/blob/master/co2eq/purchase/index.js#L51 only adds children... It means that we end up with a "Food" entry without any `intensityKilograms`. 🤔

Reply:
I would expect `getEntryByKey('Food')` to return an entry with a `intensityKilograms`... Not sure if this is a regression from https://github.com/tmrowco/bloom-contrib/pull/488 or it is working as intended.

Reply:
Uh, so it wouldn't be able to run the model with a "parent" identifier?

Reply:
Works in production right now despite #488 being merged 🤔 

![image](https://user-images.githubusercontent.com/3296643/102789516-8c271080-43a4-11eb-8969-550810cca78e.png)

Reply:
Hm. 

It seems like the purchase model is doing a smarter lookup for the entry and carbon intensity than doing `getEntryByKey('Food')`.

Reply:
I've verified that the activity you linked to, has the `Food` as `identifier`. And the `carbonModelName` is `purchase` version 9.

Reply:
Turned out to be a Jest caching issue: `yarn test --clearCache` did the job. Thanks @madsnedergaard !